R/write_dataset_json.R

```diff
@@ -65,18 +80,22 @@ write_dataset_json <- function(x, file, pretty=FALSE) {
         }
         x[y$name] <- strftime(as.numeric(x[[y$name]]), "%H:%M:%S", tz='UTC')
       }
+    } else if (float_as_decimals && y$dataType == "float") {
+      meta$columns[[i]]['dataType'] <- "decimal"
```

Comment:
Should we actually take on setting these to decimal, or leave it to sponsors to decide if they set dataType and targetDataType to decimal?

Reply:
My intent of the option is to forcefully set floats to decimal, which is why it defaults to FALSE. The docs state that this is exactly what happens so it's a conscious choice. What I want to happen here is that we convert both the data and the metadata. The vignette also explains what's happening and why to use it.